ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -62,13 +60,30 @@
  * @author athomas
  */
 public class GanttCSVExport {
+
+  public enum FileExtensionEnum {
+    CSV() {
+      @Override
+      public String toString() {
```

Comment:
GanttCSVExport:
```
  public enum Format {
    CSV("csv"), XLS("xls");
    private final String myExtension;
    Format(String extension) {
      myExtension = extension;
    }
    @Override
    public String toString() {
      return "impex.csv.fileFormat" + name().toLowerCase();
    }
    public String getExtension() {
      return myExtension;
    }
  }
```

ExporterToCSV:
```
  static class FormatOption extends DefaultEnumerationOption<GanttCSVExport.Format> {
    FormatOption() {
      super("impex.csv.format", GanttCSVExport.Format.values());
      setSelectedValue(GanttCSVExport.Format.CSV);
    }
  }
  private final FormatOption myFormatOption = new FormatOption();

  public String proposeFileExtension() {
    return myFormatOption.getSelectedValue().getExtension();
  }
```

i18n.properties (didn't try but I think this is right):
```
optionValue.impex.csv.fileFormat.csv.label = CSV
optionValue.impex.csv.fileFormat.xls.label = XLS
```